Makefile

```diff
@@ -0,0 +1,25 @@
+SHELL := /bin/bash -e
+
+ifneq ($(wildcard .env),)
```

Comment:
Hey @DominicGunn Just part of the standard makefile we use everywhere.  That way you can share them between your makefile and your app :)